### ISSUE TITLE
How to Display Notifications

### ISSUE THREAD

**Issue by jasminechau2** [vocatwk/CodeU-Nemo#65]
I was wondering how the notifications should be displayed. I was thinking of implementing a drop down list. ie) facebook. I am open to any other ideas though. I did find an example of how to implement a drop down menu that uses JQuery. I am not sure if we are open to using that 

Here is a link: http://www.encodedna.com/jquery/create-a-facebook-like-notifications-using-jquery-css.htm

**Comment by yebrahim** [comment]
Sometimes it would help visualize things before working on the design, see this mock I created using an online tool: https://www.fluidui.com/editor/live/preview/cF9NTGxjN3ZySmdHRVZ4bGw2c0ZMbzZ5V0JsSlJyazJOaw==
(click "Restart preview" at the top)
It's a three-page mock up. You can click the little notifications circle icon in the mock, which will transition to the next page, where you can click any item in the dropdown list to go to the next.

**Comment by jasminechau2** [comment]
This is great! It really helps to see how it could work!

**Comment by yebrahim** [comment]
Yup. Mocking UI is a great tool to measure your progress too, we should consider creating a simple mock for how we want the app to look like. That can be a separate issue though.
